presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analyzer.java

```diff
@@ -64,7 +64,7 @@ public Analysis analyze(Statement statement, boolean isDescribe)
         Statement rewrittenStatement = StatementRewrite.rewrite(session, metadata, sqlParser, queryExplainer, statement, parameters, accessControl);
         Analysis analysis = new Analysis(rewrittenStatement, parameters, isDescribe);
         StatementAnalyzer analyzer = new StatementAnalyzer(analysis, metadata, sqlParser, accessControl, session);
-        analyzer.process(rewrittenStatement, Scope.builder().markQueryBoundary().build());
+        analyzer.process(rewrittenStatement, Scope.create());
```

Comment:
shouldn't this also have parent according to: https://github.com/prestodb/presto/pull/6738/commits/d15ca4150e50ff4f9d18aad369c2132115d330bd#r91843198?